src/export/StructureDefinitionExporter.ts

```diff
@@ -71,6 +72,43 @@ export class StructureDefinitionExporter implements Fishable {
     private readonly fisher: MasterFisher
   ) {}
 
+  /**
+   * Checks generated resources for any custom resources which are not in the
+   * http://hl7.org/fhir/StructureDefinition namespace, and are therefore not
+   * comformant.
+   */
+  private warnOnNonComformantResourceDefinitions(): void {
```

Comment:
Minor typo here, and in a few other spots in this function:
```suggestion
  private warnOnNonConformantResourceDefinitions(): void {
```

Reply:
Oopsy, looks like my spelling wasn't too conformant am I right.

Reply:
Better spelling in https://github.com/FHIR/sushi/pull/859/commits/4f0e3013eae8f4d0e0651130f2501e981ec375ac.